src/pages/Dashboard/container.js

```diff
@@ -213,6 +213,30 @@ const DashboardContainer = Wrapped =>
 
       return true;
     };
+    
+    checkIfManditoryEvent = (startDate) => {
+
+      // set the manditory dates (year is never checked)
+      const mandatoryDates = ['2019-00-01', '2019-05-27', '2019-12-25'];
+
+      // Check if the startDate has the same day and month as any of the mandatory dates
+      const isMandatoryDate = mandatoryDates.some( dateString => {
```

Comment:
looks ok but how do we handle bookings across a mandatory date? e.g. starting 24th Dec to 26th December? really it should be 2 days taken from holidays - is this accounted for?

Reply:
This is now managed outside of the application. Mandatory dates can no longer be created or updated via the front end.